person.rb

```diff
@@ -5,14 +5,15 @@
 
 class Person < Nameable
```

Comment:
- [ ] It will be great to have a method for adding rentals. Remember there is a many-many relationship between person and book through the rentals. I can see you have implemented the `add-rental` in your book class, which handles the books, I can also see you have pushed the `self` into the `person.rentals` and `book.rentals`, this handles the bridge between the person and the book. However, when I check the person class we do not have a method to handle adding of rentals. kindly check the example below. ⬇️ 

```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Reply:
This change has been done well

Reply:
Thanks @peterrobert 🙏